build-system/tasks/karma.conf.js

```diff
@@ -121,9 +121,9 @@ module.exports = {
   },
 
   singleRun: true,
-  browserDisconnectTimeout: 10000,
+  browserDisconnectTimeout: 1000000,
```

Comment:
Oops, did not mean to commit this file. This change was just for debugging with Karma, so that the browser wouldn't refresh immediately while I was trying to debug a test.